server/controllers/animalReport.js

```diff
@@ -231,10 +233,33 @@ const deleteAnimalReport = async (req, res) => {
     }
 }
 
+//GET: Retrieves all animal reports by the owner ID
```

Comment:
Hey! We already have a getAnimalReportById up top. Please delete this function as it's essentially a duplicate.